config/default/core.entity_view_display.node.campground_directory_record.default.yml

```diff
@@ -77,7 +77,7 @@ mode: default
 content:
   field_at_least_one_wheelchair_ac:
     type: boolean
-    label: inline
+    label: above
```

Comment:
Please double-check whether we need these changes or not.

Reply:
It is correct according to this https://github.com/ytgov/yukon-ca/pull/51/files#diff-8cdbc8d155ee2e98dcb85f918aaaa11f4211ec9a9a6c2f4d48cb4c62192b036dR79

Reply:
reverted